iTechArt.CinemaWebApp.API/Controllers/CinemasController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using iTechArt.CinemaWebApp.API.Data;
+using iTechArt.CinemaWebApp.API.Model;
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace iTechArt.CinemaWebApp.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CinemasController : ControllerBase
+    {
+        private readonly CinemaDbContext _context;
+
+        public CinemasController(CinemaDbContext cinemaDbContext)
+        {
+            _context = cinemaDbContext;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Cinema>>> GetCinemas()
+        {
+            return await _context.Cinemas.AsNoTracking().OrderBy(cinema => cinema.Id).ToListAsync();
```

Comment:
Please format chained LINQ expressions starting each call from the new line like that: 
```suggestion
            return await _context.Cinemas
                .AsNoTracking()
                .OrderBy(cinema => cinema.Id)
                .ToListAsync();
```